package.json

```diff
@@ -25,16 +25,15 @@
   "license": "ISC",
   "homepage": "https://bitbucket.org/revolve-it/eslint-config-revolve#readme",
   "dependencies": {
-    "@react-native-community/eslint-config": "^3.2.0",
```

Comment:
RN eslint config is removed from deps because it is required from react native by default